modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -3,6 +3,8 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: api, manager, raml
+:page-aliases: api-manager:2.x:latest-overview-concept.adoc
```

Comment:
```suggestion
:page-aliases: api-manager::latest-overview-concept.adoc
```
It's not necessary to specifically determine the version for a latest version. 
Also, if you would need to specify versions, you would use a different syntax:

```
:page-aliases: 2.x@api-manager::latest-overview-concept.adoc
```

You can validate this against the page identifier docs in Antora: https://docs.antora.org/antora/2.1/page/page-id/#structure

Reply:
Perfect, thank you. I found some broken links to, so it'll take me some time to get this fixed.

Reply:
On a second review, shouldn't this page-aliases be?
```
:page-aliases: general:getting-started:api-lifecycle-overview.adoc
```